en/6-use-open-standards-solutions.md

```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
 
 <section class="dpgn-section-guideline">
 
-## 6.1 Leverage open standards and embrace leading practices
+## 6.1 Leverage open standards, use open source software and embrace leading practices
 
 <div class="dpgn-section-intro-guideline">
 
 **[TODO: Add/revise introductory text]**
 
+Give equal consideration to open source software when you choose technology.

```

Comment:
This sounds like an action that should be taken. Is it already covered adequately in the checklist items?